build-utils.js

```diff
@@ -94,41 +97,85 @@ function taskHarness(task, projectOrStar, ...args) {
  * also add a license header and create sourcemaps.
  *
  * @param  {Object} options Options object with 'projectDir' and 'rollupConfig'
- * @return {[type]}         [description]
+ * @return {Promise}
  */
 function buildJSBundle(options) {
-  const destPath = path.join(options.projectDir, 'build');
-  const licenseHeader = fs.readFileSync('LICENSE-HEADER', 'utf8');
+  return new Promise((resolve, reject) => {
+    const outputName = options.buildPath.replace(/^build\//, '');
+    const sources = [
+      `${options.projectDir}/src/**/*.js`,
+      '{lib,packages}/**/*.js',
+      'node_modules/*/**/*.js',
```

Comment:
Just tried changing it, and you get a

```
[10:59:25] Starting 'build'...

events.js:160
      throw er; // Unhandled 'error' event
      ^
Error: Input files must be buffered!
```

if you don't explicitly use the original glob pattern. I think without that original pattern, `gulp.src` picks up the symlinks in `node_modules/.bin` and gets confused?

Anyway, I'm assuming that's why you had it set up that way. I'll add a comment.